src/components/@shared/DebugOutput/index.tsx

```diff
@@ -9,7 +10,16 @@ export default function DebugOutput({
 }): ReactElement {
   return (
     <div style={{ marginTop: 'var(--spacer)' }}>
-      {title && <h5>{title}</h5>}
+      <div
+        style={{
+          display: 'flex',
+          justifyContent: 'space-between',
+          flexWrap: 'wrap'
+        }}
+      >
+        {title && <h5>{title}</h5>}
+        {output && <DDODownloadButton asset={output} />}
```

Comment:
issue(blocking): the download button should be available, even if the user is not in Debug-Mode